src/Compilers/CSharp/Portable/Generated/BoundNodes.xml.Generated.cs

```diff
@@ -562,32 +562,36 @@ public BoundDisposableValuePlaceholder Update(TypeSymbol type)
 
     internal sealed partial class BoundObjectOrCollectionValuePlaceholder : BoundValuePlaceholderBase
     {
-        public BoundObjectOrCollectionValuePlaceholder(SyntaxNode syntax, TypeSymbol type, bool hasErrors)
+        public BoundObjectOrCollectionValuePlaceholder(SyntaxNode syntax, bool isNew, TypeSymbol type, bool hasErrors)
             : base(BoundKind.ObjectOrCollectionValuePlaceholder, syntax, type, hasErrors)
         {
 
             RoslynDebug.Assert(type is object, "Field 'type' cannot be null (make the type nullable in BoundNodes.xml to remove this check)");
 
+            this.IsNew = isNew;
         }
 
-        public BoundObjectOrCollectionValuePlaceholder(SyntaxNode syntax, TypeSymbol type)
+        public BoundObjectOrCollectionValuePlaceholder(SyntaxNode syntax, bool isNew, TypeSymbol type)
             : base(BoundKind.ObjectOrCollectionValuePlaceholder, syntax, type)
         {
 
             RoslynDebug.Assert(type is object, "Field 'type' cannot be null (make the type nullable in BoundNodes.xml to remove this check)");
 
+            this.IsNew = isNew;
         }
 
 
         public new TypeSymbol Type => base.Type!;
+
+        public bool IsNew { get; }
         [DebuggerStepThrough]
```

Comment:
Unrelated to this PR: Does it worth to update the code that generates this to leave an empty line after this property?

Reply:
Meh, we have lots of functional issues. Formatting generated code isn't worth spending time :-/